.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -87,7 +87,7 @@ jobs:
           make "${KATA_ASSET}-tarball"
           build_dir=$(readlink -f build)
           # store-artifact does not work with symlink
-          sudo cp -r "${build_dir}" "kata-build"
+          mkdir -p kata-build && cp "${build_dir}"/kata-static-${KATA_ASSET}*.tar.* kata-build/.
```

Comment:
Hi @zvonkok @stevenhorsman !

Yes, it does work. Thanks!

There is a situation where it might copy the tarball more than one time, but I don't think it is a big deal. When the asset's name is sub-string of another asset's name, for example, "kernel" and "kernel-confidential":
```
$ rm -rf kata-build/
$ export KATA_ASSET=kernel
$ build_dir=$(pwd)/build ; mkdir -p kata-build && cp "${build_dir}"/kata-static-${KATA_ASSET}*.tar.* kata-build/.
$ ls kata-build/
kata-static-kernel-confidential-modules.tar.xz  kata-static-kernel-confidential.tar.xz  kata-static-kernel.tar.xz
$ rm -rf kata-build/
$ export KATA_ASSET=kernel-confidential
$ build_dir=$(pwd)/build ; mkdir -p kata-build && cp "${build_dir}"/kata-static-${KATA_ASSET}*.tar.* kata-build/.
$ ls kata-build/
kata-static-kernel-confidential-modules.tar.xz  kata-static-kernel-confidential.tar.xz
```

We were already copying everything over and over again, this is just a corner case, again, not a big deal IMO.